### DISCUSSION TITLE
Cannot ping outbound network in EC2 machine

### DISCUSSION THREAD

**Discussion by tal-rofe98** [Genymobile/feedback#166] [Genymotion Device Image] [answered]
Hi, I'm using this AMI in the marketplace in AWS EC2:  `Genymotion Cloud - Android 11.0 (arm64)`

I successfully set up the machine, and then browsed to its web console. Then I entered the `shell` and tried to ping `8.8.8.8` but got an error:
```
connect: Network is unreachable
```

I don't understand why. Also, this is my EC2 configuration (default) for network:
<img width="794" alt="image" src="https://github.com/user-attachments/assets/5c516b7d-853a-417b-9b92-aa857291d073">

**Comment by jmaneyrol69** [comment, marked as answer]
Hi @tal-rofe98,

I am terribly sorry but this public discussion board is not appropriate for technical assistance.
Please contact our technical support team at https://support.genymotion.com/hc/en-us/requests/new

Thank you.
